### ISSUE TITLE
Consider using HAR format for HTTP type tests

### ISSUE THREAD

**Issue by hellais** [TheTorProject/ooni-spec#93]
There is no reason to re-invent the wheel and as we may need to support more detailed information related to loading of web pages (for example parsing javascript and like) we may consider switching to using HAR instead: https://w3c.github.io/web-performance/specs/HAR/Overview.html.

It appears to be JSON based so it would fit nicely within our data format.

**Comment by bassosimone** [comment]
Sounds reasonable! 👍

**Comment by bassosimone** [comment]
Yes, I believe that we should. I did a pass at reading the HAR spec. It's possible to use this spec to represent OONI data. We would need some `_` prefixed fields to keep the extra stuff we care about, though. Therefore, I am not sure whether we should use HAR or generate HAR from OONI data.

**Comment by bassosimone** [comment]
I have been working on this as part of https://github.com/ooni/probe-engine/issues/12.

**Comment by bassosimone** [comment]
I believe doing this is clearly a good idea, because it would allow to load / analyse requests using a bunch of standard tools including Chrome Dev tools.
